src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_build_format.py

```diff
@@ -0,0 +1,175 @@
+
+
+def _format_group(item):
+    """Returns an ordered dictionary of the container registry.
+    :param dict item: The container registry object
+    """
+    table_info = {_property_map[key]: str(
+        item[key]) for key in item if key in _property_map}
+
+    try:
+        import json
+        table_info['TASK'] = json.loads(item['buildTask'])['BuildTaskName']
+    except (KeyError, TypeError):
+        table_info['TASK'] = item['buildType']
+
+    try:
+        table_info['PLATFORM'] = item['platform']['osType']
+    except (KeyError, TypeError):
+        pass
+
+    table_info['STARTED'] = _get_start(item['startTime'])
+
+    try:
+        table_info['DURATION'] = _get_difference(
+            item['finishTime'], item['startTime'])
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['SKU'] = item['sku']['name']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['USERNAME'] = item['username']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['PASSWORD'] = item['passwords'][0]['value']
+    except (KeyError, TypeError, IndexError):
+        pass
+
+    try:
+        table_info['PASSWORD2'] = item['passwords'][1]['value']
+    except (KeyError, TypeError, IndexError):
+        pass
+
+    try:
+        # Only show ID if it is not an ARM resource ID
+        table_info['ID'] = item['id'] if '/subscriptions/' not in item['id'].lower() else None
+    except (KeyError, TypeError):
+        pass
+
+    # Parse webhook list-events
+    try:
+        table_info['ACTION'] = item['eventRequestMessage']['content']['action']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['IMAGE'] = item['eventRequestMessage']['content']['target']['repository']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        tag = item['eventRequestMessage']['content']['target']['tag']
+        if table_info['IMAGE'] and tag:
+            table_info['IMAGE'] = '{}:{}'.format(table_info['IMAGE'], tag)
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['TIMESTAMP'] = item['eventRequestMessage']['content']['timestamp']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        table_info['RESPONSE STATUS'] = item['eventResponseMessage']['statusCode']
+    except (KeyError, TypeError):
+        pass
+
+    try:
+        status_code = item['eventResponseMessage']['statusCode']
+        reason_phrase = item['eventResponseMessage']['reasonPhrase']
+        table_info['RESPONSE STATUS'] = '{} {}'.format(
+            status_code, reason_phrase)
+    except (KeyError, TypeError):
+        pass
+
+    return OrderedDict(sorted(table_info.items(), key=lambda t: _order_map[t[0]]))
+
+
+def _get_difference(finishTime, startTime):
+    from datetime import datetime
+    from dateutil import relativedelta
+    import dateutil.parser
+    try:
+        date_start = dateutil.parser.parse(startTime)
```

Comment:
I am not too familiar with this method. Can we possibly use methods from just `datetime` such as for this one, use `datetime.strptime`?